### PR TITLE
test treasury/branch

### PR DESCRIPTION
**Pull request by Define101**

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/treasury/psyoptions.js: 
        
 
  
    
    
    TypeError: Cannot read properties of undefined (reading 'value')
    Truncated error stack:
    
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/solana.js:157:43
        at Array.forEach (<anonymous>)
        at getTokenBalances (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/solana.js:157:22)
        at async _sumTokens (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/solana.js:407:27)
        at async Object.sumTokens2 (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/solana.js:383:7)
        at async getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:53:23)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:149:9
        at async Promise.all (index 1)
        at async Promise.all (index 0)

File: projects/treasury/psyoptions.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+module.exports = treasuryExports({
+  solana: {
+    owners: [
+      'CcNUW7KDCdaUY6rNqYJBmTKYn66RjYTVyPUqCNEiALdp',
+      'CyDnoEMVuf21v23bxoS2wXxPdCvRR2yFLfymegMH1WY4',
+      'E4tuwwYvmB9XWKTmhxeRmywhMmttyDLtMok6jQVAbDLG'
+    ],
+    ownTokens: [
+      '2v6gYajLxSd2xdfRMz8LCUJrc79Z26PTGmVzqLA5g6KW', // PSY
```
Comment (g1nt0ki) [inline projects/treasury/psyoptions.js:11]: this is a token account, the token is: PsyFiqqjiv41G7o5SMRzDJCu4psptThNR2GtfeGHfSq

https://solscan.io/token/PsyFiqqjiv41G7o5SMRzDJCu4psptThNR2GtfeGHfSq

**Comment by llamatester** [comment]
Error while running adapter at projects/treasury/psyoptions.js: 
        
 
  
    
    
    AxiosError: Request failed with status code 429

**Comment by llamatester** [comment]
The adapter at projects/treasury/psyoptions.js exports TVL: 
        
 
 
    solana-ownTokens          10.97 M
    solana                    1.01 M
    ownTokens                 10.97 M
    
    total                    1.01 M

**Pull request merged**